src/core/space.jl

```diff
@@ -1,5 +1,5 @@
 export node_neighbors, find_empty_nodes, pick_empty, has_empty_nodes, get_node_contents,
-id2agent, NodeIterator, space_neighbors, nodes, get_node_agents
+NodeIterator, space_neighbors, nodes, get_node_agents
```

Comment:
It doesn't make sense to deprecate something without exporting it :D otherwise users will get "Undef" error!

Reply:
Op sorry, I take this back! `@deprecate` exports automatically!

Reply:
Yep! That's the point&mdash;keeps things clean so we can just drop one line later if we want.